src/main/java/com/oneops/secrets/command/Console.java

```diff
@@ -0,0 +1,30 @@
+package com.oneops.secrets.command;
+
+import static com.oneops.secrets.utils.Color.warn;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.regex.Pattern;
+
+public class Console {
+
+  static final String INPUT = "^y(es)?$";
+
+  static void readConsole(String message) {
+    Pattern p = Pattern.compile(INPUT);
+    String in;
+    if (null != System.console()) {
+      in = System.console().readLine(warn(message));
+    } else {
+      try {
```

Comment:
Can remove the `try catch` here also, as the exception is handled upstream (same as `System.console().readLine`)